src/compareTheNumbers.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { terminal } = require('./terminal');
+const { findBullsAndCows } = require('./findBullsAndCows');
+const isNumber = require('is-number');
+
+function compareTheNumbers(message, randomNumber) {
+  terminal.question(message, (num) => {
+    const isLengthCorrect = num.length === 4;
```

Comment:
so here when you check the length you also need to check the uniqueness of all four digits.